Rawl_B512/ferchuitred_medba.trig

```diff
@@ -0,0 +1,431 @@
+@base <http://example.com/Rawl_B512/ferchuitred_medba.trig> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix rel: <http://purl.org/vocab/relationship/>.
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
+@prefix irishRel: <http://example.com/earlyirishRelationship.ttl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix irishTitles: <http://example.com/earlyIrishTitles.ttl#> .
+
+<http://example.com/Rawl_B512> {
+  <> a dctype:Dataset ;
+        dcterms:title "Ferchuitred Medba"@sga ;
+        dcterms:isFormatOf <https://celt.ucc.ie/published/G301010/text001.html> ;
+        dcterms:format "application/trig" ;
+        prov:asDerivedFrom <https://celt.ucc.ie/published/G301010/text001.html> .
+
+  <#EochaidFedhlech>
+        a foaf:Person ;
+        irishRel:nomName "Eochaid Fedhlech" ;
+        rel:childOf <#Find>;
+        owl:sameAs <http://example.com/Rawl_B502/mínigud_na_cróeb_coibnesta_inso_sís.trig#EochuFeidlech-e8f620e5>.
+
+  <#Find>
+        a foaf:Person ;
+        irishRel:genName "Find" ;
+        rel:childOf <#RoighenRuaid>.
+
+  <#RoighenRuaid>
+        a foaf:Person ;
+        irishRel:genName "Roighen Ruaid" ;
+        rel:descendantOf <#RitfathaScuit>.
+
+  <#RitfathaScuit>
+        a foaf:Person ;
+        irishRel:genName "Ritfatha Scuit" .
+
+  <#Cessair>
+        a foaf:Person ;
+        irishRel:nomName "Cessair" .
+# sameAs <http://example.com/Rawl_B502/_94.trig#CessairCáemchruthaich> ? - CGY
+# seems unlikely as Cessair Cáemchruthaich is female
+
+  <#EochaidFedhlech-eba6d350>
+        owl:sameAs <#EochaidFedhlech>;
+        irishRel:numChild 3 .
+# MS 23 P 2 says 4 sons - CGY http://www.ucd.ie/tlh/text/jon.eriu.2.001.text.html
+
+  <#Bres>
+        a foaf:Person ;
+        irishRel:nomName "Bres" ;
+        rel:childOf <#EochaidFedhlech-eba6d350>;
+        rel:parentOf <#LugaidSriab>;
+        owl:sameAs <http://example.com/Rawl_B502/mínigud_na_cróeb_coibnesta_inso_sís.trigBress> .
```

Comment:
Yes, I know about this. If I change it and push, you will not be able to see the entire file.